### ISSUE TITLE
Suddenly Doesn't Work with Command Palette (quick-input-widget)

### ISSUE THREAD

**Issue by dtomasch** [drcika/apc-extension#74]
When the plugin was first released, I used it to modify the command palette to look more like the old Atom CP.

On the last launch, this doesn't seem to work.

I have tried both the css as an import and directly in the user json — but the quick-input-widget never seems to be affected; this did work previously with the plugin and with the earlier customize UI and Monkey Patch.

Any thoughts? I can provide any and all further details.

**Comment by drcika** [comment]
check css selectors, maybe they changed something

![Screenshot 2023-09-04 at 03 59 16](https://github.com/drcika/apc-extension/assets/35919606/4d380d96-1404-49a5-8ae0-457318bddfda)

**Comment by dtomasch** [comment]
It looks like the actual problem is the CSS isn't being applied, as opposed to anything wrong with the code itself.

I ran the chown per the directions in the FAQ and copied over your code above — no change.

I deleted the contents of my user json and deactivated all extensions except this one, and still no change.

It worked just a few weeks ago

<img width="1399" alt="Screen Shot 2023-09-04 at 10 46 00 PM" src="https://github.com/drcika/apc-extension/assets/11246228/89322af5-9bd4-4815-9d49-98a3d9eb216f">


Version: 1.81.1
Commit: 6c3e3dba23e8fadc360aed75ce363ba185c49794
Date: 2023-08-09T22:40:25.698Z (3 wks ago)
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 21.6.0
